tests/test_cli_compile.py

```diff
@@ -23,11 +23,34 @@
 
 @pytest.fixture
 def pip_conf(tmpdir, monkeypatch):
+    test_conf = dedent(
+        """\
+        [global]
+        no-index = true
+        find-links = {wheels_path}
+        """
+    )
+
+    pip_conf_file = "pip.conf" if os.name != "nt" else "pip.ini"
+    path = (tmpdir / pip_conf_file).strpath
+
+    with open(path, "w") as f:
+        f.write(test_conf.format(wheels_path=MINIMAL_WHEELS_PATH))
+
+    monkeypatch.setenv("PIP_CONFIG_FILE", path)
+
+    try:
+        yield path
+    finally:
+        os.remove(path)
+
+
+@pytest.fixture
+def pip_with_index_conf(tmpdir, monkeypatch):
```

Comment:
I'd prefer to create a helper fixture to make code more DRY:
```python
@pytest.fixture
def make_pip_conf(tmpdir, monkeypatch):
    created_paths = []

    def _make_pip_conf(content):
        pip_conf_file = "pip.conf" if os.name != "nt" else "pip.ini"
        path = (tmpdir / pip_conf_file).strpath

        with open(path, "w") as f:
            f.write(content)

        monkeypatch.setenv("PIP_CONFIG_FILE", path)

        created_paths.append(path)
        return path

    try:
        yield _make_pip_conf
    finally:
        for path in created_paths:
            os.remove(path)
```
and then create needed fixtures by:

```python
@pytest.fixture
def pip_conf(make_pip_conf):
    return make_pip_conf(
        dedent(
            """\
            [global]
            no-index = true
            find-links = {wheels_path}
            """.format(
                wheels_path=MINIMAL_WHEELS_PATH
            )
        )
    )

@pytest.fixture
def pip_with_index_conf(make_pip_conf):
    return make_pip_conf(
        dedent(
            """\
            [global]
            index-url = http://example.com
            find-links = {wheels_path}
            """.format(
                wheels_path=MINIMAL_WHEELS_PATH
            )
        )
    )
```

Reply:
@atugushev Good advice. It is Done!